### PR TITLE
[POAE7-1175] add API for APE adoption

### PR DESCRIPTION
**Pull request by yma11**
add write(ByteBuf byteBuf, int off, int len) API for APE adoption.

### CODE REVIEW

**Comment by yma11** [comment]
@xieqi please take a review.

File: src/main/java/com/intel/compression/spark/IntelCompressionCodecBlockOutputStream.java [hunk 1]
```diff
@@ -21,11 +21,15 @@
 import java.io.OutputStream;
 import java.io.FilterOutputStream;
 import java.nio.ByteBuffer;
+
+import com.intel.compression.util.Platform;
+import io.netty.buffer.ByteBuf;
```
Comment (xieqi) [inline src/main/java/com/intel/compression/spark/IntelCompressionCodecBlockOutputStream.java:26]: do we need add netty dependency in pom?

**Comment by xieqi** [comment]
@yma11 LGTM

File: src/main/java/com/intel/compression/spark/IntelCompressionCodecBlockOutputStream.java (hunk 1 above)
Comment (yma11) [reply src/main/java/com/intel/compression/spark/IntelCompressionCodecBlockOutputStream.java:26]: netty dependency should included in the ones of Spark.

**Pull request merged**